internal/pkg/service/buffer/storage/statistics/repository/schema_test.go

```diff
@@ -17,61 +16,77 @@ type keyTestCase struct{ actual, expected string }
 func TestSchema(t *testing.T) {
 	t.Parallel()
 	s := newSchema(serde.NewJSON(serde.NoValidation))
-
-	time1, _ := time.Parse(time.RFC3339, "2006-01-02T15:04:05+07:00")
-	time2 := time1.Add(time.Hour)
-
-	projectID := keboola.ProjectID(123)
-	receiverKey := key.ReceiverKey{ProjectID: projectID, ReceiverID: "my-receiver"}
-	exportKey := key.ExportKey{ExportID: "my-export", ReceiverKey: receiverKey}
-	fileKey := key.FileKey{ExportKey: exportKey, FileID: key.FileID(time1)}
-	sliceKey := key.SliceKey{SliceID: key.SliceID(time2), FileKey: fileKey}
-	nodeID := "my-node"
+	sliceKey := test.NewSliceKey()
 
 	cases := []keyTestCase{
 		{
-			s.InCategory(Buffered).Prefix(),
-			"stats/buffered/",
+			s.InLevel(storage.LevelLocal).Prefix(),
+			"storage/stats/local/",
+		},
+		{
+			s.InLevel(storage.LevelStaging).Prefix(),
+			"storage/stats/staging/",
+		},
+		{
+			s.InLevel(storage.LevelTarget).Prefix(),
+			"storage/stats/target/",
 		},
 		{
-			s.InCategory(Uploaded).Prefix(),
-			"stats/uploaded/",
+			s.InLevel(storage.LevelLocal).InProject(sliceKey.ProjectID).Prefix(),
+			"storage/stats/local/123/",
 		},
 		{
-			s.InCategory(Imported).Prefix(),
-			"stats/imported/",
+			s.InLevel(storage.LevelLocal).InReceiver(sliceKey.ReceiverKey).Prefix(),
+			"storage/stats/local/123/my-receiver/",
 		},
 		{
-			s.InCategory(Buffered).InReceiver(receiverKey).Prefix(),
-			"stats/buffered/123/my-receiver/",
+			s.InLevel(storage.LevelLocal).InReceiver(sliceKey.ReceiverKey).Sum().Key(),
+			"storage/stats/local/123/my-receiver/_sum",
 		},
 		{
-			s.InCategory(Buffered).InExport(exportKey).Prefix(),
-			"stats/buffered/123/my-receiver/my-export/",
+			s.InLevel(storage.LevelLocal).InExport(sliceKey.ExportKey).Prefix(),
+			"storage/stats/local/123/my-receiver/my-export/",
 		},
 		{
-			s.InCategory(Buffered).InExport(exportKey).CleanupSum().Key(),
-			"stats/buffered/123/my-receiver/my-export/_cleanup_sum",
+			s.InLevel(storage.LevelLocal).InExport(sliceKey.ExportKey).Sum().Key(),
+			"storage/stats/local/123/my-receiver/my-export/_sum",
 		},
 		{
-			s.InCategory(Buffered).InFile(fileKey).Prefix(),
-			"stats/buffered/123/my-receiver/my-export/2006-01-02T08:04:05.000Z/",
+			s.InLevel(storage.LevelLocal).InFile(sliceKey.FileKey).Prefix(),
+			"storage/stats/local/123/my-receiver/my-export/2000-01-01T19:00:00.000Z/",
 		},
 		{
-			s.InCategory(Buffered).InSlice(sliceKey).Prefix(),
-			"stats/buffered/123/my-receiver/my-export/2006-01-02T08:04:05.000Z/2006-01-02T09:04:05.000Z/",
+			s.InLevel(storage.LevelLocal).InSlice(sliceKey).Key(),
+			"storage/stats/local/123/my-receiver/my-export/2000-01-01T19:00:00.000Z/my-volume/2000-01-01T20:00:00.000Z/value",
```

Comment:
Hlavny rozdiel v scheme je, ze nakonci je `/value`, kod ktorym to aggregujeme je potom jednoduchsi, kedze aj pre poslednu uroven - `SliceKey`, mozem pouzit prefix a nemusis to riesit inak.